### PR TITLE
updating urls for new service mount location

### PR DESCRIPTION
**Pull request by jhjaggars**

### CODE REVIEW

**Comment by beav** [comment]
In https://github.com/RedHatInsights/insights-upload/pull/117#issuecomment-474976963, @dehort wisely suggested these be set as env vars since we are changing them anyway.

Something like this should do the trick

```patch
diff --git a/app.py b/app.py
index 60b9299..658f596 100644
--- a/app.py
+++ b/app.py
@@ -94,6 +94,11 @@ VALIDATION_QUEUE = os.getenv('VALIDATION_QUEUE', 'platform.upload.validation')
 
 INVENTORY_URL = os.getenv('INVENTORY_URL', 'http://inventory:8080/api/hosts')
 
+PATH_PREFIX = os.getenv('PATH_PREFIX', '/api/')
+APP_NAME = os.getenv('APP_NAME', 'ingress')
+
+API_PREFIX = PATH_PREFIX + APP_NAME
+
 # Message Queue
 MQ = os.getenv('KAFKAMQ', 'kafka:29092').split(',')
 MQ_GROUP_ID = os.getenv('MQ_GROUP_ID', 'upload')
@@ -524,9 +529,9 @@ class MetricsHandler(NoAccessLog):
 
 
 endpoints = [
-    (r"/r/insights/platform/upload", RootHandler),
-    (r"/r/insights/platform/upload/api/v1/version", VersionHandler),
-    (r"/r/insights/platform/upload/api/v1/upload", UploadHandler),
+    (API_PREFIX, RootHandler),
+    (API_PREFIX + "/v1/version", VersionHandler),
+    (API_PREFIX + "/v1/upload", UploadHandler),
     (r"/metrics", MetricsHandler)
 ]
```

File: app.py [hunk 1]
```diff
@@ -524,9 +524,9 @@ def get(self):
 
 
 endpoints = [
-    (r"/r/insights/platform/upload", RootHandler),
-    (r"/r/insights/platform/upload/api/v1/version", VersionHandler),
-    (r"/r/insights/platform/upload/api/v1/upload", UploadHandler),
+    (r"/api/ingress/", RootHandler),
```
Comment (beav) [inline app.py]: minor nitpick: new url has trailing /, old one does not. I don't know if this URL is actually used or not.

**Comment by jhjaggars** [comment]
@beav I'll close this one, since you have suggested a better solution.  Do you have a PR ready?

**Comment by beav** [comment]
> @beav I'll close this one, since you have suggested a better solution. Do you have a PR ready?

i do, it fails a couple of tests but those may be related more to my env than the code change.

My old change kept it as `/upload`, i'll re-open as `/ingress`

**Comment by beav** [comment]
@jhjaggars https://github.com/RedHatInsights/insights-upload/pull/122

**Comment by SteveHNH** [comment]
Replaced by #122

**Pull request closed without merging**